docs/get-started/_index.md

```diff
@@ -1,36 +1,165 @@
+title: "Learn about Viam"
+linkTitle: "Learn about Viam"
 weight: 5
-type: "docs"
+description: "Viam is a complete software platform for smart machines which provides modular components and services for vision, motion, SLAM, ML, and data management."
 no_list: true
+type: docs
+aliases:
+  - "/getting-started/"
+  - "/getting-started/high-level-overview"
+  - "/product-overviews/"
+  - "/viam/"
+  - "/viam/app.viam.com/"
+imageAlt: "/general/understand.png"
+images: ["/general/understand.png"]
+carouselscript: true
 ---
 
-Learn about the Viam platform by reading our Viam in 3 minutes guide or try it out for yourself:
+Viam is a software platform that makes it easy to combine and integrate hardware and software to build machines, connect them with the cloud, and make them smarter with machine learning.
 
-{{< cards >}}
-{{% card link="/get-started/viam/" %}}
-{{% card link="/get-started/try-viam/" %}}
-{{% card link="/get-started/quickstarts/control-motor/" %}}
-{{% card link="/get-started/quickstarts/collect-data/" %}}
-{{% card link="/get-started/quickstarts/detect-people/" %}}
-{{% card link="/get-started/quickstarts/write-code/" %}}
-{{% card link="/get-started/quickstarts/drive-rover/" %}}
-{{< /cards >}}
+Machines are everywhere, from small machines like IoT sensors, to home automation systems, robotic systems, cars and boats, and even more complex enterprise systems.
+All these machines start by combining hardware and software.
 
-If you're ready to try it on your own machine, start by installing `viam-server`.
-If you need any help along the way, join our community:
+<div>
+{{< imgproc src="/viam/viam.png" alt="Viam overview" resize="800x" class="aligncenter" >}}
+</div>
 
-<!-- markdownlint-disable-file MD034 -->
+## `viam-server` and the Viam app
 
-{{< cards >}}
-{{% card link="/get-started/installation/" %}}
-{{% manualcard link="https://discord.gg/viam" webm_src="/heart.webm" mp4_src="/heart.mp4" alt="A robot drawing a heart" %}}
+At the core of Viam is the open-source `viam-server` executable which runs on a computer and manages hardware, software, and data for a machine.
+To use Viam with a machine, you create a configuration specifying which hardware and software the machine consists of.
+`viam-server` then manages and runs the drivers for the configured hardware components and software services.
+
+For example, if you are building a pet-feeding machine you might install `viam-server` on a Raspberry Pi and create a machine configuration with a camera and a servo for the hardware you are using.
+
+If you are working with microcontrollers, the [micro-RDK](/installation/) is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`.
+
+The [Viam app](https://app.viam.com) provides the user interface for configuring your machine.
+You can put together any of the available resources across microcontrollers and computers as configurable building blocks.
```

Comment:
This sorta sounds like the microcontrollers and computers are the building blocks. I think "available" might also throw people.
```suggestion
You can put together any components and services as configurable building blocks.
```